internal/image/build.go

```diff
@@ -97,16 +97,20 @@ func (a *attestation) NewSignOffSource() (signOffSource, error) {
 // get the last commit used for the component build
 func (a *attestation) getBuildCommitSha() string {
 	if len(a.Predicate.Materials) == 1 {
-		return a.Predicate.Materials[0].Digest["sha"]
+		fmt.Printf("Using commit sha: %s", a.Predicate.Materials[0].Digest["sha1"])
```

Comment:
This should use the log (import `log "github.com/sirupsen/logrus"`):
```suggestion
		log.Debug("Using commit sha: %s", a.Predicate.Materials[0].Digest["sha1"])
```